src/scripts/modules/add_destination.js

```diff
@@ -30,24 +30,11 @@ function bindEvents() {
     setValues();
     addDestination();
     event.preventDefault();
+    const itemIndex = destinationsList.length;
```

Comment:
`length - 1 `, otherwise the index is invalid. Maybe that's why you were having the error.

Do you understand why the `length - 1` or you need me to explain it in detail? :)

Reply:
Omg yes it was perfectly clear, my brain just doesn't work. I'm gonna go sort some vowels on codewars or something for a couple days 😆